dialoguekit/utils/dialogue_reader.py

```diff
@@ -69,14 +69,18 @@ def json_to_annotated_utterance(
 
 
 def json_to_dialogues(
-    filepath: str, agent_id: str, user_id: str
+    filepath: str,
+    agent_id: str,
+    user_id: str,
+    conversation_id: str,
```

Comment:
This method parse a JSON containing multiple dialogues each having a conversation_id, so the argument `conversation_id` should be retrieved from the JSON data.

Reply:
Done